script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -498,7 +498,7 @@ So there's really no need for you to worry.
 [sha:76be4af800aa8bf80162af74eb364f2bba5717c0]{
 -- Page 186, Offset 30043.
 -- それにあの夜だって―――あんなに苦しそうだったのに、ちゃんと我慢できたじゃないか。だから大丈夫だよ。これからも今までみたいにやっていける」
-And that night―――even though it was agonizing, you were able to endure it, weren't you?
+And that night―――even though it was agonizing, you were able to endure it, weren't you.
```

Comment:
for all of these . -> ?, unless they're driven by the VA, revert. Ross last touched these files and he tends to base punctuation on the character's intonation

Reply:
Yeah if these changes are driven by the times I've commented that not every question needs a ?, there's no need to swing too hard the opposite direction.